lego/apps/content/models.py

```diff
@@ -55,19 +55,25 @@ class Content(SlugModel):
 
     slug_field = "title"
 
-    @property
-    def reactions_grouped(self):
+    def get_reactions_grouped(self, user):
```

Comment:
I suspect this will be kinda non-optimal, as it will result in a few more queries than needed. Not so important now, but I think most of the logic can be done in sql instead of python

Reply:
Agreed, I just went built on what was previously created. I will consider moving the logic if I have the time, or fix it in a future PR.